File: src/java/Boot/src/com/core/api/mcpe/World.java
package com.core.api.mcpe;

import com.core.api.mcpe.level.LevelData;

public class World {
    native public static void setFlatGenerator(String json);
    native public static void addWorldToCache(String world_path);
    native public static void updateWorlds();

  
    native public static int getWorldsCount();
    native private static long _getLevelData(String maybe_uid);

    native public static void createLevel(String uid,String path);
    public static LevelData getLevelData(String uid){
        return new LevelData(_getLevelData(uid));
    }
}

File: src/dev/api/World.js
let WorldJavaAPI = WRAP_JAVA("com.core.api.mcpe.World");

World.addWorldToCache = function(path){
    return WorldJavaAPI.addWorldToCache(path);
};

World.updateWorlds = function(){
    return WorldJavaAPI.updateWorlds();
};

World.getWorldsCount = function(){
    return WorldJavaAPI.getWorldsCount();
};

World.setFlatGenerator = function(json){
    return WorldJavaAPI.setFlatGenerator(json);
};

World.setFlatGenerator('{"biome_id":1,"block_layers":[{"block_data":0,"block_id":7,"count":1},{"block_data":0,"block_id":3,"count":2},{"block_data":0,"block_id":2,"count":1}],"encoding_version":3,"structure_options":null}')

